simulation_parameters/SimulationParameters.cs

```diff
@@ -177,17 +178,16 @@ public bool DoesCompoundExist(string name)
     /// </summary>
     public List<Compound> GetCloudCompounds()
     {
-        var result = new List<Compound>();
-
-        foreach (var entry in compounds)
-        {
-            if (entry.Value.IsCloud)
-            {
-                result.Add(entry.Value);
-            }
-        }
+        return compounds.Values.Where(c => c.IsCloud).ToList();
+    }
 
-        return result;
+    /// <summary>
+    ///   Returns all environmental *molecular* compounds that are dissolved in the environment, i.e. gas.
```

Comment:
I feel like this comment should be a bit reworded to be just "Returns compounds that are dissolved in the environment (aren't clouds)" just so that the comment won't become incorrect in the future.